include/ace/generic/main.h

```diff
@@ -50,14 +50,12 @@ int main(void) {
 	blitManagerCreate();
 	copCreate();
 
-	gameCreate();
```

Comment:
Since you're removing them from generic `main()`, this is a breaking change. I'm fine with that, but sample code in docs needs updating to reflect those changes. Or, perhaps the generic main should create the first instance of game manager, since most of the games will use it anyway.

Reply:
I've created generic global state manager.